src/processing/board_object/BoardObjectOperationType.java

```diff
@@ -0,0 +1,24 @@
+package processing.board_object;
```

Comment:
Not a fan of underscore character. What does the coding convention recommend for multiple words: boardobject or board_object?

Reply:
You are watching older file, I don't know how but it's changed already.

Reply:
You changed the commit and viewing an older file.